lib/AST/NameLookupRequests.cpp

```diff
@@ -363,7 +363,7 @@ void LookupInModuleRequest::writeDependencySink(
 }
 
 //----------------------------------------------------------------------------//
-// LookupConformanceInModuleRequest computation.
+// AnyObjectLookupRequest computation.
 //----------------------------------------------------------------------------//
 
 void swift::simple_display(llvm::raw_ostream &out,
```

Comment:
This overload of simple_display() is for LookupConformanceInModuleRequest though. Maybe you want to add a new comment on line 379?

Reply:
sure makes sense. I just figured the comment intended to refer to what came later, but it was copy and paste duplication that was never modified, but I will move it proximate.